extensions/wikia/AdEngine/js/video/player/porvata/floater.js

```diff
@@ -40,9 +43,7 @@ define('ext.wikia.adEngine.video.player.porvata.floater', [
 				elements.video.addEventListener('start', listeners.start);
 			}
 
-			if (elements.video.isPlaying()) {
-				elements.video.resize(width, height);
-			}
+			elements.video.resize(width, height);
```

Comment:
Because we want to resize video event when its paused. Not doing that ends with not resized floating video.